manifests/init.pp

```diff
@@ -77,6 +81,11 @@
     Exec { timeout => $timeout }
   }
 
+  if $cookies {
+    validate_array($cookies)
```

Comment:
The module has had its last puppet 3 release. This means you can use puppet 4 data types instead of validate functions and I think this is preferred.